keanu-project/src/main/java/io/improbable/keanu/network/NetworkLoader.java

```diff
@@ -5,8 +5,12 @@
 import io.improbable.keanu.vertices.dbl.DoubleVertex;
 import io.improbable.keanu.vertices.intgr.IntegerVertex;
 
+import java.io.IOException;
+import java.io.InputStream;
+
 public interface NetworkLoader {
 
+    BayesianNetwork loadNetwork(InputStream input) throws IOException;
```

Comment:
Nit: might be nice to add a tiny bit of whitespace between this (which is the "global load" function) and the value load functions below?

Reply:
Haha, "a tiny bit of whitespace" :) Added/